content/en/Platform Deep Dive/Collaboration/groups.md

```diff
@@ -16,19 +16,22 @@ Watch a demo video going over the Groups feature:
 
 ## Create a Group
 
-Assign assets and [Organization Members](/platform-deep-dive/collaboration/user-roles/#organization-member) into a group:
+Assign [Organization Members](/platform-deep-dive/collaboration/user-roles/#organization-member) into a group:
 
 - Navigate to the **People** page, and then to the **Groups** tab.
 - Select **Create Group**.
 - The **Create Group** screen prompts you for the following:
   - **Group Name**: Set up a descriptive name to easily identify the group.
   - **Description**: Add information to further describe the group.
-  - **Associated Assets**: Select from a list of all assets within your organization (Note: Assets already associated with another group will not appear in this list).
   - **Members**: Select from a list of all Organization Members within your organization. These users will have exclusive access to the group’s associated assets, and their pentests and findings.
     - Organization Owners and Cobalt Staff will not appear in this list but will still have access to all assets.
     - [Pentest Team Members](/platform-deep-dive/collaboration/user-roles/#pentest-team-member) that are not part of your organization can still be [manually added](/platform-deep-dive/collaboration/manage-collaborators/#add-a-pentest-team-member) to each pentest.
 - Select **Create Group**.
 
+{{% alert title="Note" color="primary" %}}
+Assets can be [individually assigned](/collaboration/groups/#assigning-a-group-to-an-asset) to a group from the Create or Edit Asset forms.
```

Comment:
I don't know why the line breaks here.

Reply:
It is still one line, it wraps because there's not enough space.

Reply:
<img width="1726" alt="Screenshot 2024-05-15 at 07 49 37" src="https://github.com/cobalthq/cobalt-product-public-docs/assets/102162593/cdae9d17-d926-4c7a-841e-bbb1563d38f9">
Hmm ok. It still looks like I have space, though.

Reply:
Ohh I thought you meant in the PR preview 😅 . For some reason I see 2 links in the rendered HTML. 
<img width="530" alt="Screenshot 2024-05-15 at 17 35 40" src="https://github.com/cobalthq/cobalt-product-public-docs/assets/25580776/b9ed08f7-9c2a-4727-b544-0ae65ec3ba69">

Do we know if there are other examples of links inside a note 🤔?

Reply:
<img width="969" alt="Screenshot 2024-05-15 at 08 11 56" src="https://github.com/cobalthq/cobalt-product-public-docs/assets/102162593/1cbd1ec4-1313-4854-a7f0-fcec032e1b75">

there's an example in `platform-deep-dive/collaboration/manage-collaborators/#remove-a-pentest-team-member`